step.yml

```diff
@@ -130,8 +130,20 @@ inputs:
         Prefer to do a manual `git merge` by default.
         When the Pull Request is from a GitHub or Bitbucket private fork repository set this to `no`.
       value_options:
-        - "yes"
-        - "no"
+      - "yes"
+      - "no"
+  - fetch_tags: automatic
+    opts:
+      category: Clone Config
+      title: Should the step fetch tags?
+      descriptions: |-
+        automatic - fetches all tags from the remote in case of a tag checkout, otherwise fetches tags that point at objects that are downloaded from the remote repository
```

Comment:
automatic - fetches all tags from the remote repository in the case of a tag checkout, otherwise fetches tags that point at objects that are downloaded from the remote repository. (Did you mean "fetches all tags from the remote **repository**? I added it to the sentence)